samples/Grafana/README.md

```diff
@@ -0,0 +1,55 @@
+### Was this documentation helpful? [Share feedback](https://www.research.net/r/DGDQWXH?src=documentation%2Flocalmachine)
```

Comment:
nit: Update the `src` here, looks like it got copy-pasted from our `localmachine` docs